src/v2/Apps/Show/ShowApp.tsx

```diff
@@ -24,9 +31,61 @@ export const ShowApp: React.FC<ShowAppProps> = ({ show }) => {
         <HorizontalPadding>
           <ShowInstallShots show={show} my={2} />
 
-          <Text as="h1" variant="largeTitle" my={4}>
-            {show.name}
-          </Text>
+          <Grid>
+            {!hasAbout && !hasViewingRoom && (
+              <Col span={[12, 8, 6]}>
+                <ShowHeader show={show} />
+              </Col>
+            )}
+
+            {hasAbout && !hasViewingRoom && (
+              <>
+                <Col span={6}>
+                  <ShowHeader show={show} />
+                </Col>
+
+                <Col span={6}>
+                  <ShowAbout show={show} />
+                </Col>
+              </>
+            )}
+
+            {!hasAbout && hasViewingRoom && (
+              <>
+                <Col span={6}>
+                  <ShowHeader show={show} />
+                </Col>
+
+                <Col span={1} display={["none", "block"]} />
+
+                <Col span={5}>
+                  <ShowViewingRoom />
+                </Col>
+              </>
+            )}
+
+            {hasAbout && hasViewingRoom && (
+              <>
+                <Col span={[12, 8, 6]}>
+                  <ShowHeader show={show} />
+                </Col>
+
+                <Break />
+
+                <>
+                  <Col span={6}>
+                    <ShowAbout show={show} />
+                  </Col>
+
+                  <Col span={1} display={["none", "block"]} />
+
+                  <Col span={5}>
+                    <ShowViewingRoom />
+                  </Col>
+                </>
+              </>
+            )}
```

Comment:
The Figma spec for this broke my brain. There's a different layout for each state of these two booleans.